src/modules/App/MobileLayout.tsx

```diff
@@ -1,7 +1,7 @@
 import './mobile.scss';
 
 import React, { useState, useEffect } from 'react';
-import type { BaseMessage } from '@sendbird/chat/message';
```

Comment:
Could you revert this change?

Reply:
`BaseMessage` is used in line 57, not by this PR. `UserMessage` is removed.